lib/ClangImporter/ClangModuleDependencyScanner.cpp

```diff
@@ -265,17 +265,28 @@ ModuleDependencyVector ClangImporter::bridgeClangModuleDependencies(
       swiftArgs.push_back(IncludeTree);
     }
 
+    // Ensure we get the "real", vfs-remapped path to the modulemap
+    // becuase it is used as an explicit input for the resulting compilation
+    // task.
+    SmallString<128> realModuleMapPath =
+        getClangInstance()
+            .getFileManager()
```

Comment:
Is this guaranteed to be configured with exactly the same VFS the Clang scanner gets?

Reply:
Yeah, they are instantiated from the exact same set of arguments.